### DISCUSSION TITLE
Webhook untuk Pesan Masuk

### DISCUSSION THREAD

**Discussion by rioastamal** [kirimwa-id/forum#7] [Roadmap]
Mengirim webhook ketika pesan baru masuk. Dengan adanya fitur ini maka developer dapat membangun sendiri fitur chatbot atau auto reply.

Vote up dan beri tanda 👍  &nbsp; jika fitur ini ingins segera diimplementasikan.

**Comment by rioastamal** [comment]
Fitur ini telah diimplementasikan untuk pesan bertipe teks. Lihat pengumuman di #32.

**Comment by Septian-Bhoechie** [comment]
wah mantep iki

**Reply by rioastamal** [reply]
Ayo dicoba! 😎

**Comment by agitnaeta** [comment]
Aku menambahkanyah, untuk incoming message saya pernah dapat request biar punya nama sender nya automatis  a.k.a pengirim nya
Di beberapa sistem checkout pengen nama pelanggan bisa langsung terisi saat ada yang chat bisa langsung jadi akun pelanggan / data pelanggan

**Reply by rioastamal** [reply]
@agitnaeta terima kasih infonya, kedepan akan dipelajari.

**Comment by rioastamal** [comment]
Tipe pesan yang akan diimplementasi kedepan:
- [ ] Gambar
- [ ] Video
- [ ] File
- [ ] Contact
- [ ] Location
- [ ] Live Location

**Comment by akbarism** [comment]
webhook tipe ini bisa di disable ga mas? atau allow sendernya dimasukin nomor fake aja? h3h3
